index.bs

```diff
@@ -268,33 +279,40 @@ xrDevice.requestSession({ immersive: true }).then((session) => {
 Session {#session}
 =======
 
-XRSessionCreationOptions {#xrsessioncreationoptions-interface}
+XRSessionMode {#xrsessionmode-enum}
 -------------------------
 
 <pre class="idl">
-dictionary XRSessionCreationOptions {
-  boolean immersive = false;
-  XRPresentationContext outputContext;
+enum XRSessionMode {
+  "inline",
+  "immersive-vr",
+  "immersive-ar"
 };
 </pre>
 
-The {{XRSessionCreationOptions}} dictionary provides a <dfn>session description</dfn>, indicating the desired properties of a session to be returned from {{requestSession()}}.
+The {{XRSessionMode}} enum defines the modes that an {{XRSession}} can operate in.
 
-<div class="algorithm" data-algorithm="supported-by-device">
+  - A session mode of <dfn enum-value for="XRSessionMode">inline</dfn> indicates that the session's output will be shown as an element in the HTML document. {{inline}} session content MAY be displayed in mono or stereo and MAY allow for [=viewer=] tracking. User agents MUST allow {{inline}} sessions to be created for any [=XR device=].
+  - A session mode of <dfn enum-value for="XRSessionMode">immersive-vr</dfn> indicates that the session's output will be shown on the [=XR device=] hardware in a manner that occupies a significant portion of the user's vision. The {{environmentBlendMode}} MAY be {{opaque}} for {{immersive-vr}} sessions.
```

Comment:
> The {{environmentBlendMode}} MAY be {{opaque}} for {{immersive-vr}} sessions.

I'm inclined to see this be a bit stronger a statement.  For example, we don't want to allow {{alpha-blend}}, right?  We might want to take it further and say the only time {{additive}} should be allowed is if the hardware cannot support {{opaque}}.